source/includes/steps-stop-sharded-cluster.yaml

```diff
@@ -0,0 +1,34 @@
+title: "Stop :binary:`mongos <bin.mongos>` routers."
+stepnum: 1
+ref: stop-mongos
+content: |
+
+   Run :method:`db.shutdownServer()` from the ``admin`` database on each
+   :binary:`mongos <bin.mongos>` router:
+
+   .. code-block:: javascript
+
+      use admin
+      db.shutdownServer()
+
+---
+title: "Stop shards."
+stepnum: 2
+ref: stop-shards
+content: |
+
+   Run :method:`db.shutdownServer()` from the ``admin`` database on each
```

Comment:
Users will have to start with the secondaries, then finish on the primary. The primary *should* step down on its own once a majority of members are offline (I think). That way I think we avoid having to talk about `rs.stepDown(<stepdownsecs.)` or potentially `rs.stepDown({force : true})`

Reply:
done